### DISCUSSION TITLE
Adding two-way interactive directions between Alarms Table, TreeMap, and Time Series

### DISCUSSION THREAD

**Discussion by moawnallah** [InternetHealthReport/gsoc#25] [General]
@romain-fontugne  I'd like to add the following interactions `Tree Map <-> Alarms Table (Row Selected) <-> Time Series (Legend Selected)`. This is related to [Updating other graph when selecting AS or country in the treemap](https://github.com/orgs/InternetHealthReport/projects/10/views/1?pane=issue&itemId=37675301).

**Comment by romain-fontugne** [comment]
yes, I think for users it's more intuitive to have all views synced together. This would be a really nice improvement
